### PR TITLE
Correct user-facing misspellings

### PR DESCRIPTION
**Pull request by foster33**
The following PR serves to correct misspellings throughout Accumulo that users may see.

Corrections were found in log messages, exceptions, print statements, etc.

### CODE REVIEW

File: core/src/main/java/org/apache/accumulo/core/file/BloomFilterLayer.java [hunk 1]
```diff
@@ -544,7 +544,7 @@ public static void main(String[] args) throws IOException {
 
     t2 = System.currentTimeMillis();
 
-    out.printf("existant lookup rate %6.2f%n", 500 / ((t2 - t1) / 1000.0));
+    out.printf("existent lookup rate %6.2f%n", 500 / ((t2 - t1) / 1000.0));
```
Comment (EdColeman) [inline core/src/main/java/org/apache/accumulo/core/file/BloomFilterLayer.java:547]: Maybe this should be existing?
``` suggest
    out.printf("existing lookup rate %6.2f%n", 500 / ((t2 - t1) / 1000.0));
```

File: core/src/main/java/org/apache/accumulo/core/util/AddressUtil.java [hunk 2]
```diff
@@ -61,7 +61,7 @@ public static int getAddressCacheNegativeTtl(UnknownHostException originalExcept
     try {
       negativeTtl = Integer.parseInt(Security.getProperty("networkaddress.cache.negative.ttl"));
     } catch (NumberFormatException exception) {
-      log.warn("Failed to get JVM negative DNS respones cache TTL due to format problem "
+      log.warn("Failed to get JVM negative DNS responses cache TTL due to format problem "
```
Comment (EdColeman) [inline core/src/main/java/org/apache/accumulo/core/util/AddressUtil.java:64]: Maybe response?
``` suggest
      log.warn("Failed to get JVM negative DNS response cache TTL due to format problem "
```

Comment (DomGarguilo) [inline core/src/main/java/org/apache/accumulo/core/util/AddressUtil.java:64]:
```suggestion
      log.warn("Failed to get JVM negative DNS response cache TTL due to format problem "
```

**Review by DomGarguilo — CHANGES_REQUESTED** [review]

**Review by EdColeman — APPROVED** [review]
Mainly looked for changes that would lead to errors - I'm not the best speller so saying those changes look correct is not much of an endorsement, but other than a few questions on word choice - I do not see other issues.

**Review by DomGarguilo — APPROVED** [review]

**Review by EdColeman — APPROVED** [review]
LGTM - same caveats to my spelling apply though.

**Pull request merged**